bin/test_tex_math.py

```diff
@@ -0,0 +1,233 @@
+from lxml import etree
+from anthology.texmath import TexMath
+
+texmath = TexMath()
+
```

Comment:
Some things that might be nice to add (all taken from our data):

- ```<tex-math>0(n^{\tilde{\rho}+1})</tex-math>``` to have one more complex expression
- ```<tex-math>\{mt, src\} \rightarrow pe</tex-math>``` because I'd like to make sure we handle the `\{`s right
- ```<tex-math>p(\boldsymbol{y}|\textrm{do}(\boldsymbol{x}))</tex-math>``` to test `\boldsymbol` and `\textrm`
- ```<tex-math>{\sim}3\%</tex-math>``` for a use case of curly brackets that _I believe_ tests the hacks I mentioned above, because `{\sim}` is not the argument of a function here